packages/flutter/test/rendering/editable_test.dart

```diff
@@ -409,58 +409,6 @@ void main() {
     expect(editable, paintsExactlyCountTimes(#drawRect, 1));
   });
 
-  test('ignore key event from web platform', () async {
```

Comment:
Do we still have tests to verify this? Or why is this no longer needed?

Reply:
I removed this because RenderEditable no longer handles key events at all.  Now that I look at this again though, it's probably useful to test that web doesn't trigger the text editing actions.  I'll bring this back in a way that covers that.

Reply:
It's now in EditableText's test, modified a bit.